### ISSUE TITLE
Store Location: Profiler - Step 1

### ISSUE THREAD

**Issue by verofasulo** [woothemes/woocommerce#32585]
## User Story

> As a new merchant going through the onboarding process,
> I want to see choices relevant to my country,
> To be able to set up my store faster and without having to see irrelevant details

## Acceptance Criteria

```
Feature: Onboarding Profiler

  Scenario: User starts step 1 of onboarding wizard
    When the user starts the onboarding wizard
    Then subheading should be "Tell us where you run your business to help us configure currency, shipping, taxes, and more in a fully automated way." without a tooltip.
    And the fields should appear in the order of Country / Region, Address, Post / Zip Code, City, Email Address.
    And the Country/Region is a mandatory field marked with an asterisk
    And the "Continue" button should appear below the fields.
    But is disabled unless the mandatory field (Country / Region) is completed.

  Scenario: Mandatory fields are completed
    When the user completes the mandatory field (Country / Region)
    Then the "Continue" button is enabled.
```

**Comment by chihsuan** [comment]
Hi @pmcpinto or @SamratBiswas1, @ilyasfoo has created an wonderful issue template for explat (pdibGW-kU-p2#comment-388). Would you be able to create an Explat issue for the store location profiler based on that? It would be very helpful for us before we start working on this in the next cycle. Thanks. 🙇‍♂️

**Comment by pmcpinto** [comment]
@chihsuan we don't have a way to run an experiment in this step because users only start sharing data with us after the click "continue" or "skip".
I'd keep this change simple and move forward with variant 1. @verofasulo @SamratBiswas1 let me know if you agree

**Comment by SamratBiswas1** [comment]
@pmcpinto - I agree that we should keep it simple. However, I am leaning towards variant 3 for the following reasons:
1. The country information will help us better serve the users with country specific extensions during onboarding
2. email will allow us to communicate and keep the conversation going incase they drop off durign the process.
 
I see both the information useful in user's onbaording journey. Seems like our competition mandates it too.

@verofasulo - Over to you!

**Comment by chihsuan** [comment]
>  we don't have a way to run an experiment in this step because users only start sharing data with us after the click "continue" or "skip".

@pmcpinto Ah, right. That would be a problem. 🤔

**Comment by verofasulo** [comment]
I agree with @SamratBiswas1 - My original idea was to test all the 3 designs to split the variables and have more accurate data. Since we can't, I would lean toward a bolder approach and I would test the 3rd option, even if more variables are combined here. One of my assumptions that I'd love to test is that users skip this step because they are intimidated by this wall of mandatory fields as a starting point.

**Comment by pmcpinto** [comment]
@SamratBiswas1 @verofasulo thanks for the feedback. Just to clarify: the goal is to make the country field as mandatory even if users decide to skip the profiler, right?

> Reorder the fields: Country/Region, Postcode, Address, City, Email address.

I think I never saw the postcode displayed before the address. I think it's better to invert the order: Country/Region, Address, Postcode...

**Comment by SamratBiswas1** [comment]
Yes. Country and email mandatory.

**Comment by pmcpinto** [comment]
@SamratBiswas1 thanks for the clarification. I'd start with just country mandatory. The reason is that we only track users after they continue or skip this initial step. Suppose the numbers go down after making these changes, we don't have a way to tell if the drop is caused by the changes or just a slowdown in demand. And I'm afraid that we're adding too much friction if we have 2 mandatory fields. Makes sense?
@verofasulo let me know what you think as well.

**Comment by verofasulo** [comment]
@pmcpinto I think it's a great idea - it allows us to have more accurate data.

**Comment by SamratBiswas1** [comment]
Hi @verofasulo - Does that mean we need to create a variant 4 with only country as mandatory? and then go ahead and implement it?

**Comment by adrianduffell** [comment]
> Share information about your store later

@verofasulo in issue #32587, we'll streamline the "Store details" Task and remove the ability to go back to the profiler. Would we need a new way for users to return to the profiler?

**Comment by verofasulo** [comment]
> Does that mean we need to create a variant 4 with only country as mandatory? and then go ahead and implement it?

@SamratBiswas1 yes

> Would we need a new way for users to return to the profiler?

@adrianduffell I don't think it's needed. They are 2 different parts of the experience and apart from adding store location and choosing a theme, I don't see why users should go back to fill out information like industry and type of products. What do you think?

**Comment by adrianduffell** [comment]
> They are 2 different parts of the experience and apart from adding store location and choosing a theme, I don't see why users should go back to fill out information like industry and type of products. What do you think?

@verofasulo  Yeah, I'd agree with that. My initial thoughts were "share information about your store later" might mislead the user into thinking they could come back to the profiler later. On second thoughts, I can see we'll give them the opportunity to share those more important items later via the task list.

**Comment by verofasulo** [comment]
> My initial thoughts were "share information about your store later" might mislead the user into thinking they could come back to the profiler later

That's a great point @adrianduffell - What about changing the copy into "Skip"? 
Generally speaking, I think we should change the approach but it's something we can do later when we rethink the signup experience. I think we should differentiate between steps, as right now the user can just skip or complete the profiler entirely - there's no possibility to make some steps mandatory and allow the users to skip others.

**Comment by SamratBiswas1** [comment]
@verofasulo @adrianduffell - I think 'Skip' is an universally understood. I would be agree with it.
Furthermore, I would like to have a 'Skip' option on all the profiler pages. It gives a user to 'Skip' anytime they want. If the user doesn't want to provide the info then we lose them completely and so an option of 'Skip' will allow us to move them to product task

**Comment by adrianduffell** [comment]
"Skip" also sounds good to me!

**Comment by adrianduffell** [comment]
@verofasulo @SamratBiswas1 Can you confirm we should remove the marketing checkbox in this?

```
Get tips, product updates and inspiration straight to your mailbox.
```

**Comment by adrianduffell** [comment]
I've updated the description with the changes discussed in the comments so far, and converted the issue to User Story + Gherkin format.

**Comment by adrianduffell** [comment]
Also have now extracted the skip button changes into a new issue https://github.com/woocommerce/woocommerce/issues/34063.

**Comment by adrianduffell** [comment]
Also removed the screenshots because they became a bit out of date 😄

**Comment by chihsuan** [comment]
Hey @verofasulo 👋  I have a question.

In the current design, we use the same store details component in the shipping task, so when we change the form in OBW, it will also change the shipping task. Do you think it's okay to update the shipping task with the new design? Or we should keep using the current one.

Current:
![Screen Shot 2022-08-02 at 14 40 28](https://user-images.githubusercontent.com/4344253/182309159-2ce81fcf-4dc5-4302-8c04-68f14a3eca94.png)

New:
![Screen Shot 2022-08-02 at 14 40 57](https://user-images.githubusercontent.com/4344253/182309181-7e27b7f9-61b4-4f06-8830-d9368fb482bd.png)

**Comment by verofasulo** [comment]
> Do you think it's okay to update the shipping task with the new design? Or we should keep using the current one.

Excellent point @chihsuan 🙂 I think it's ok to align the shipping form, it makes total sense to me.
